universum/nonci.py

```diff
@@ -16,14 +16,17 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     def execute(self):
-
         self.out.log("Cleaning artifacts...")
         self.artifacts.clean_artifacts_silently()
 
         project_configs = self.process_project_configs()
+        afterall_configs = self.code_report_collector.prepare_environment(project_configs)
         self.artifacts.set_and_clean_artifacts(project_configs, ignore_existing_artifacts=True)
 
         self.launch_project()
+        if afterall_configs:
+            self.code_report_collector.report_code_report_results()
```

Comment:
What is the purpose of this line?
There is no vcs and no code review system in nonci mode.

Reply:
If I got it right, this is the function, that transforms intermediate files into final report in artifacts. I might be wrong, though.

Reply:
That is true, but who is supposed to use this report?

Reply:
test

Reply:
test

Reply:
test

Reply:
test